easybuild/easyconfigs/f/Fiji/Fiji-20240605-1417.eb

```diff
@@ -0,0 +1,51 @@
+easyblock = 'PackedBinary'
+
+name = 'Fiji'
+version = '20240605-1417'
+
+homepage = 'https://fiji.sc/'
+description = """Fiji is an image processing package—a 'batteries-included' distribution of
+ ImageJ, bundling a lot of plugins which facilitate scientific image analysis.
+This release is based on ImageJ-2.1.0 and Fiji-2.1.1"""
+
+toolchain = SYSTEM
+
+source_urls = ['https://downloads.imagej.net/%(namelower)s/archive/%(version)s']
+sources = [{
+    'download_filename': '%(namelower)s-nojre.zip',
+    'filename': '%(namelower)s-nojre-%(version)s.zip',
+}]
+checksums = ['d9329f79f548db1d74143146215b02f18a005549e57847cf8df3bedb2905ab12']
+
+dependencies = [('Java', '1.8', '', SYSTEM)]
```

Comment:
1.8 is the old Oracle java, please avoid using it. Based on earlier PRs, at least JAva 11 should work for this software. So, maybe 11 or 17?